internal/commands/build_test.go

```diff
@@ -126,6 +126,32 @@ func testBuildCommand(t *testing.T, when spec.G, it spec.S) {
 					h.AssertContains(t, outBuf.String(), "Builder 'heroku/builder:24' is trusted")
 				})
 			})
+
+			when("the builder is known to be trusted but not suggested", func() {
+				it("sets the trust builder option", func() {
+					mockClient.EXPECT().
+						Build(gomock.Any(), EqBuildOptionsWithTrustedBuilder(true)).
+						Return(nil)
+
+					logger.WantVerbose(true)
+					command.SetArgs([]string{"image", "--builder", "heroku/builder:22"})
+					h.AssertNil(t, command.Execute())
+					h.AssertContains(t, outBuf.String(), "Builder 'heroku/builder:22' is trusted")
+				})
+			})
+
+			when("the builder is not trusted", func() {
+				it("warns the user that the builder is untrusted", func() {
+					mockClient.EXPECT().
+						Build(gomock.Any(), EqBuildOptionsWithImage("org/builder:unknown", "image")).
```

Comment:
I think you're right @edmorley. I tried `EqBuildOptionsWithTrustedBuilder(false)` initially but the assertion would always fail. Debugging through that assertion showed that it was never checking the expected value, just asserting that the builder was trusted so it would only pass if that was `true`. I've correct the matcher logic and switched back to `EqBuildOptionsWithTrustedBuilder(false)`.